site/public/js/drag-and-drop.js

```diff
@@ -715,7 +731,7 @@ function handleBulk(gradeable_id, num_pages, use_qr_codes = false, qr_prefix = "
             catch (e) {
                 alert("Error parsing response from server. Please copy the contents of your Javascript Console and " +
                     "send it to an administrator, as well as what you were doing and what files you were uploading.");
-                console.log(data);
+                console.log(e);
```

Comment:
Please revert this change. The exception here (and other places we wrap `JSON.parse`) will not really give any meaningful information, most likely just being `SyntaxError: Unexpected token a in JSON at position 0`, whereas `data` will be the frog robot or other output which will have buried in it some idea of where to start looking.